torch/jit/__init__.py

```diff
@@ -1481,6 +1481,16 @@ class ScriptModule(with_metaclass(ScriptMeta, Module)):
 
         ``ScriptModule``\s should not be created manually, instead use
         either :func:`tracing <torch.jit.trace>` or :func:`scripting <torch.jit.script>`.
+
+        * Tracing records the tensor operations as executed with a set of example inputs and uses these
+          operations to construct a computation graph. You can use the full dynamic behavior of Python with tracing,
+          but values other than Tensors and control flow aren't supported.
+
+        * Scripting inspects the Python code of the model
```

Comment:
Line breaking here is funky.

Reply:
It's some rST nonsense, it breaks the rendered spacing if there's not a break here